ElrondNetwork/api.elrond.com
# add withScamInfo query flag

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- NFTs scam info processing is CPU intensive
  
## Proposed Changes
- Add a flag in order to limit scam info display for NFT list

## How to test
- Check if /nfts does not return any scam info
- Set withScamInfo=true and check again (scam info should appear for NFTs that are scam)
